Resources/public/js/views/services/ez-contenteditviewservice.js

```diff
@@ -36,9 +36,11 @@ YUI.add('ez-contenteditviewservice', function (Y) {
             this.after('*:closeView', this._redirectAfterClose);
             this.after('discardedDraft', this._redirectAfterDiscard);
             this.after('publishedDraft', this._redirectAfterPublish);
+            this.after('contentEditView:activeChange', this._refreshSession);
```

Comment:
I think the same problem will be in `create`. Maybe it would be good to `refreshSession` in the entire application not only in edit mode?